pyproject.toml

```diff
@@ -33,7 +35,7 @@ dependencies = [
     "psutil",
     "pynvml>=12.0.0,<13.0.0a0",
 ] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit dependencies.yaml and run `rapids-dependency-file-generator`.
-dynamic = ["version", "description", "authors", "urls", "keywords"]
+dynamic = ["version", "license", "description", "authors", "urls", "keywords"]
```

Comment:
Note we now flag the `license` field as dynamic since `hatch-nodejs-version` fills it in for us